### PR TITLE
Merge handbooks

### PR DESCRIPTION
**Pull request by skywall**

### CODE REVIEW

**Comment by skywall** [comment]
Readable version https://github.com/thefuntasty/ios-handbook/tree/feature/merge-handbooks

File: android/code-style.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+|   |   |   |       |-- MainViewModel.kt
+|   |   |   |       |-- MainViewModelFactory.kt
+|   |   |   |       `-- MainViewState.kt
+|   |   |   `-- App.kt 
+|   |   `-- res/layout/activity_main.xml 
+|-- keystore
+|   |-- debug.jks
+|   |-- client.jks
+|   `-- release.jks
+|-- .gitinore
```
Comment (IntergalacticPenguin) [inline android/code-style.md]:
```suggestion
|-- .gitignore
```

File: android/new-project.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+- [ ] Enable **Automatically delete head branches** so there is no need to delete branches manually after each PR merge.
+
+- [ ] Set branch protection rules to `develop` and `master` branches exactly as shown in the following screenshot:
+
+![GitHub branch protection](../generic/attachments/GitHub_branch_protection.png)
+
+> (NOTE: Status checks (Bitrise, Danger, etc.) become visible after first created pull request. Do not forget to set them afterwards.
+
+- [ ] Create new branch `feature/PROJ-1-setup-project` where everything will be setup in next steps.
```
Comment (IntergalacticPenguin) [inline android/new-project.md]:
```suggestion
- [ ] Create new branch `feature/PROJ-1-setup-project` where everything will be set up in next steps.
```

File: generic/appcenter-how-to-add-partner-account.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
+# AppCenter - How to add access to partners
+
+1. Create a new distribution group related to the app you want add access to (if already doesn't exists). 
+Distribution group management can be found at adress https://appcenter.ms/orgs/futured/apps/app-name/distribute/distribution-groups
```
Comment (IntergalacticPenguin) [inline generic/appcenter-how-to-add-partner-account.md]:
```suggestion
Distribution group management can be found at address https://appcenter.ms/orgs/futured/apps/app-name/distribute/distribution-groups 
```

File: generic/appcenter-how-to-add-partner-account.md [hunk 4]
```diff
@@ -0,0 +1,10 @@
+# AppCenter - How to add access to partners
+
+1. Create a new distribution group related to the app you want add access to (if already doesn't exists). 
+Distribution group management can be found at adress https://appcenter.ms/orgs/futured/apps/app-name/distribute/distribution-groups 
+(`app-name` should me replaced with a real name).
```
Comment (IntergalacticPenguin) [inline generic/appcenter-how-to-add-partner-account.md]:
```suggestion
(`app-name` should be replaced with a real name). 
```

File: ios/new-project.md [hunk 5]
```diff
@@ -9,7 +9,7 @@
 
 - [ ] Set branch protection rules to `develop` and `master` branches exactly as shown in the following screenshot:
 
-![GitHub branch protection](attachments/GitHub_branch_protection.png)
+![GitHub branch protection](../generic/attachments/GitHub_branch_protection.png)
 
 - [ ] Create new branch `feature/PROJ-1-setup-project` where everything will be setup in next steps.
```
Comment (IntergalacticPenguin) [inline ios/new-project.md]:
```suggestion
- [ ] Create new branch `feature/PROJ-1-setup-project` where everything will be set up in next steps.
```

**Comment by skywall** [comment]
Closes #6

File: README.md [hunk 6]
```diff
@@ -2,14 +2,23 @@
...
-### TOC
+## Generic
 
-* [Git flow](/git-flow.md)
-* [New project checklist](/new-project.md)
-* [Architecture](/architecture.md) 
-* [Code style & project structure](/code-style.md)
-* [Release checklist](/releases.md)
+* [GitFlow](/generic/git-flow.md)
+* [AppCenter - How to add access to partners](/generic/appcenter-how-to-add-partner-account.md)
```
Comment (mkj-is) [inline README.md]:
```suggestion
* [App Center - How to add access to partners](/generic/appcenter-how-to-add-partner-account.md)
```

**Comment by skywall** [comment]
@sopta ?

**Pull request merged**